src/test/datascience/widgets/standard.vscode.test.ts

```diff
@@ -58,6 +50,8 @@ suite('Standard IPyWidget (Execution) (slow) (WIDGET_TEST)', function () {
         vscodeNotebook = api.serviceContainer.get<IVSCodeNotebook>(IVSCodeNotebook);
         kernelProvider = api.serviceContainer.get<IKernelProvider>(IKernelProvider);
         traceInfo('Suite Setup (completed)');
+        // eslint-disable-next-line local-rules/dont-use-process
+        process.env.IS_WIDGET_TEST = 'true';
```

Comment:
This ensures we add the hacky property into the metadata into the cell outputs (only added in tests)

Reply:
Don't want this env variable setup in CI for all tests, that would mean we have some garbage property in outputs that could stuff up the ipynb JSON saved & break other tests.
Again, this is temporary untill #9629